doc/source/examples/unresolved-cfd-dem/gas-solid-spouted-cylinder-bed/gas-solid-spouted-cylinder-bed.rst

```diff
@@ -0,0 +1,312 @@
+    end
+
+
+Floating Walls
+~~~~~~~~~~~~~~~~~~~
+
+When we pack the cylinder with particles, we need to keep them inside, and prevent them from falling through the small channel for inlet flow. We place the stopper at the bottom of the cylinder, which is z=0 plain. This is shown in:
+
+.. code-block:: text
+
+    subsection floating walls
+      set number of floating walls = 1
+      subsection wall 0
+        subsection point on wall
+          set x = 0
+          set y = 0
+          set z = 0
+        end
+        subsection normal vector
+          set nx = 0
+          set ny = 0
+          set nz = 1
+        end
+        set start time = 0
+        set end time   = 50
+      end
+    end
+
+
+---------------------------
+Running the DEM Simulation
+---------------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``lethe-particles`` executable is within your path, the simulation can be launched on a single processor by typing:
+
+.. code-block:: text
+  :class: copy-button
+
+  lethe-particles dem-packing-in-spouted-bed.prm
+
+or in parallel (where 8 represents the number of processors)
+
+.. code-block:: text
+  :class: copy-button
+
+  mpirun -np 8 lethe-particles dem-packing-in-spouted-bed.prm
+
+After the particles have been packed inside the square bed, we can move on to the fluid-particles simulation.
+
+
+-----------------------
+CFD-DEM Parameter File
+-----------------------
+
+The CFD simulation is to be carried out using the packed bed simulated in the previous step. Here We will focus on the modified section as well.
+
+Simulation Control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation is run for 5 s with a time step of 0.001 s. The time scheme and setting for output is shown as follows.
+
+.. code-block:: text
+
+    subsection simulation control
+      set method               = bdf1
+      set number mesh adapt    = 0
+      set output name          = result_
+      set output frequency     = 50
+      set startup time scaling = 0.6
+      set time end             = 5
+      set time step            = 0.001
+      set subdivision          = 1
+      set log precision        = 10
+      set output path          = ./output/
+    end
+
+
+
+
+Boundary Conditions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Regarding the boundary conditions we apply slip boundary condition to the wall, inlet boundary to the bottom of the channel, and outlet boundary conditon to the top of the cylinder. The following shematic describes the ID of each boundary, and positon of floating wall.
+
+.. image:: images/ID.png
+    :alt: The geometry and boundary conditions
+    :align: center
+    :name: ID
+    :height: 10cm
+
+
+we set the inlet velosity to 2.5 m/s, and don't impose the background velocity on the bottom of the cylinder like the another example. The value of beta on the outlet boundary was set to 100, which is relatively high, in order to get stable simulation.
```

Comment:
```suggestion
we set the inlet velocity to 2.5 m/s, and we do not impose the background velocity on the bottom of the cylinder as in the previous spouted bed example. The value of beta on the outlet boundary was set to 100, which is relatively high, to stabilize the simulation.
```

Reply:
Why? I would maybe provide a brief explanation here or refer the reader to the parameter explanation at the parameter section.